packages/i18nify-js/src/modules/currency/__tests__/mocks/formatNumberToParts.ts

```diff
@@ -0,0 +1,1167 @@
+const nbsp = String.fromCharCode(160);
+
+export const positiveNumberPartsIntlMap = {
```

Comment:
Why do we have to name this positiveNumber ? 
We don't have any test config for negativeNumber

Reply:
We do have mocks for 'negativeNumberPartsIntlMap'